core/templates/css/oppia.css

```diff
@@ -919,6 +919,13 @@ textarea {
   font-size: 20px;
   padding-top: 13px;
 }
+.oppia-navbar-breadcrumb li {
```

Comment:
What does the li specifies? Also if this is occuring on this page it is possible that the issue might be occuring on other pages as well. Can we make it consistent for the all the pages that might be facing the same issue.(i.e. the pages that are using oppia-navbar-breadcrumb )

Reply:
@kshitij01042002 
1. `li` is inside a class called` oppia-navbar-breadcrumb`, so in order to target the `li` specifically, I used that.
2. As the style is given in the `oppia.css` file, it means that every page using `oppia-navbar-breadcrumb` is targeted.
thanks .